moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -49,48 +49,56 @@
 #include <fstream>
 #include <streambuf>
 #include <algorithm>
+#include <chrono>
+
+rclcpp::Logger LOGGER = rclcpp::get_logger("moveit").get_child("rdf_loader");
```

Comment:
this should be inside a namespace like the other loggers. 
If you wanted to use `namespace rdf_loader` then you could also find replace `rdf_loader::RDFLoader::` with `RDFLoader::` for every method in the namespace block